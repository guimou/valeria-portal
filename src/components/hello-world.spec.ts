import { createLocalVue, mount, Wrapper } from '@vue/test-utils';
import Vue, { VueConstructor } from 'vue';
import HelloWorld from './hello-world';

describe('HelloWord', () => {
    let localVue: VueConstructor<Vue>;

    beforeEach(() => {
        localVue = createLocalVue();
    });

    it('should render correctly', () => {
        expect(true).toBeTruthy();

        const rdo: Wrapper<HelloWorld> = mount(HelloWorld, {
            localVue: localVue,
            propsData: {
                msg: 'test hello world'
            }
        });

        return expect(rdo.html()).toMatchSnapshot();

    });

});
