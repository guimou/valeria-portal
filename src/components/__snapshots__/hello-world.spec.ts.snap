// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWord should render correctly 1`] = `
<div class="hello-world">
  <h1>test hello world</h1>
  <p>
    For guide and recipes on how to configure / customize this project,<br>
    check out the
    <a href="https://cli.vuejs.org" target="_blank" rel="noopener">vue-cli documentation</a>.
  </p>
  <p>
    <m-button>
      <m-icon name="edit"></m-icon> Modul button
    </m-button>
  </p>
  <p>
    <m-accordion-group class="m-u--margin-top">
      <m-accordion>
        <h2 slot="header" class="m-u--no-margin">Titre 1</h2>
        Contenu de l'accordéon 1
      </m-accordion>
      <m-accordion>
        <h2 slot="header" class="m-u--no-margin">Titre 2</h2>
        Contenu de l'accordéon 2
      </m-accordion>
      <m-accordion>
        <h2 slot="header" class="m-u--no-margin">Titre 3</h2>
        Contenu de l'accordéon 3
      </m-accordion>
      <m-accordion>
        <h2 slot="header" class="m-u--no-margin">Titre 4</h2>
        Contenu de l'accordéon 4
      </m-accordion>
    </m-accordion-group>
  </p>
</div>
`;
